File: app/src/main/java/com/example/maksim/testapp/list/model/data/GitHubUsers.java
package com.example.maksim.testapp.list.model.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by Maksim on 2017-08-28.
 */

public class GitHubUsers {
    @SerializedName("total_count")
    @Expose
    public Integer totalCount;
    @SerializedName("incomplete_results")
    @Expose
    public Boolean incompleteResults;
    @SerializedName("items")
    @Expose
    public List<GitHubUser> items = null;
}

File: app/src/main/java/com/example/maksim/testapp/list/model/data/GitHubUsers.kt
package com.example.maksim.testapp.list.model.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Maksim on 2017-08-28.
 */

class GitHubUsers {
    @SerializedName("total_count")
    @Expose
    var totalCount: Int? = null
    @SerializedName("incomplete_results")
    @Expose
    var incompleteResults: Boolean? = null
    @SerializedName("items")
    @Expose
    var items: MutableList<GitHubUser>? = null
}
